src/Wallabag/CoreBundle/Resources/translations/messages.oc.yml

```diff
@@ -375,30 +375,30 @@ howto:
         go_config: Anar a la config
         go_import: Anar per importar
         go_developers: Anar al canton desvolopaires
-        go_howto: Anar a l'ajuda (aquesta quita pagina !)
+        go_howto: "Anar a l'ajuda (aquesta quita pagina !)"
         go_logout: Desconnexion
         list_title: Acorchis disponibles dins las paginas de lista
         search: Afichar lo formulari de recèrca
         article_title: Acorchis disponibles dins la vista article
-        open_original: Dobrir l'URL originala de l'article
-        toggle_favorite: Cambiar l'estatut Favorit per l'article
-        toggle_archive: Cambiar l'estatut Legit per l'article
+        open_original: "Dobrir l'URL originala de l'article"
+        toggle_favorite: "Cambiar l'estatut Favorit per l'article"
+        toggle_archive: "Cambiar l'estatut Legit per l'article"
         delete: Suprimir l'article
         material_title: Acorchis solament disponibles amb lo tèma Material
         add_link: Apondre un acorchi
         hide_form: Rescondre lo formulari actual (recèrca o nòu ligam)
         arrows_navigation: Navigar dins los articles
-        open_article: Afichar l'article seleccionat
+        open_article: "Afichar l'article seleccionat"
```

Comment:
Double quote aren’t necessary, you can remove them

Reply:
Won't the quote cause issues?

Reply:
Oh maybe